datasources/src/main/build.sh

```diff
@@ -23,7 +23,7 @@ if [ "$RELEASE" = true ]; then
 fi
 
 # We need the gulpfile in current working directory
-cp ../../../plugins-common/gulpfile.js .
+cp ../../../plugins-common/gulpfile.mjs .
 
```

Comment:
I don't understand why this _build.sh_ file isn't a copy of the file in plugin-common ?!

Reply:
probably was the first done and not migrated to a generic one ?

You may delete it and use the generic one instead